apps/cookbook/src/app/examples/reorder-list-example/reorder-list-example.component.ts

```diff
@@ -72,7 +72,7 @@ export class ReorderListExampleComponent {
     ev.complete(ev.parentItem.shadowAccounts);
   }
 
-  trackByFn(index, item) {
+  trackByFn(_index, item) {
```

Comment:
Not used by any trackBy in component markup. Consider removing completely.

Reply:
👍  good catch